server/plugin/utils_test.go

```diff
@@ -114,3 +117,32 @@ func Test_validateTime(t *testing.T) {
 		})
 	}
 }
+
+func Test_IsCharCountSafe(t *testing.T) {
+	p := Plugin{}
+	for _, test := range []struct {
+		description  string
+		marshalError error
+	}{
+		{
+			description: "result is returned successfully with no error",
+		},
+		{
+			description:  "result is returned successfully with an error",
```

Comment:
why is it designed this way that both `result` and `error` are returned?
or may be we can make it more clear with `result is returned as false with an error`